Update Windows instructions

spotz opened the pull request.

Commit:
legacy compat

spotz committed:
Update installation instructions for Windows

Updated the instructions for installing on Windows and separated
out instructions for Linux for clarity.

spotz committed:
Update installation instructions for Windows

Updated the instructions for installing on Windows and separated
out instructions for Linux for clarity.

dietertaotesting commented on User Guide 3.2/Introduction/installing-tao.md:
```diff
@@ -1,32 +1,35 @@
 
-# Installing TAO
+ # Installing TAO
 
```
remove dot in front of hash

spotz committed:
Update installation instructions for Windows

Updated the instructions for installing on Windows and separated
out instructions for Linux for clarity.
